test/distributed/org/apache/cassandra/distributed/test/RepairTest.java

```diff
@@ -182,4 +197,28 @@ public void testParallelRepairWithoutCompression() throws Exception
     {
         repair(cluster, false, "{'enabled': false}");
     }
+
+    @Test
+    public void testForcedNormalRepairWithOneNodeDown() throws Exception
+    {
+        closeCluster();
+        int rf = 2;
+        try (ICluster testCluster = create(config -> {}, 3, rf))
+        {
+            forceRepair(testCluster, 3); // shutdown node 3 after inserting
+            DistributedRepairUtils.assertParentRepairSuccess(testCluster, 1, KEYSPACE, "test", row -> {
+                Set<String> successfulRanges = row.getSet("successful_ranges");
+                Set<String> requestedRanges = row.getSet("requested_ranges");
+                Assert.assertNotNull("Found no successful ranges", successfulRanges);
+                Assert.assertNotNull("Found no requested ranges", requestedRanges);
+                Assert.assertEquals("Requested ranges count should equals to replication factor", rf, requestedRanges.size());
+                Assert.assertTrue("Given clusterSize = 3, RF = 2 and 1 node down in the replica set, it should yield only 1 successful repaired range.",
+                                  successfulRanges.size() == 1 && !successfulRanges.contains("")); // the successful ranges set should not only contain empty string
+            });
+        }
+        finally
+        {
+            setupCluster();
```

Comment:
Would it be more efficient to just `startup()` node 3 instead?

Reply:
The cluster created for this test has a different RF. I do not want to change the existing tests, so that having a stop/start cluster wraps the actual test.